client/post-editor/editor-revisions-list/item.jsx

```diff
@@ -73,8 +77,24 @@ class EditorRevisionsListItem extends PureComponent {
 
 EditorRevisionsListItem.propTypes = {
 	revision: PropTypes.object.isRequired,
-	selectRevision: PropTypes.func.isRequired,
+	siteId: PropTypes.number.isRequired,
+
+	// connected to state
```

Comment:
We are trying to alphabetize all objects in our codebase but suddenly this breaks it. I've never seen this artificial categorization anywhere else in the app - is that some recent update to coding standards or completely new idea?

Reply:
I'm not really against but it seems to break other rules we use for reviewing the code style.

Reply:
I've started to see a lot more cases of this so started to copy... I found it useful when starting to shift towards reduxifying props since you can clearly see where they come from.
I don't feel strongly for or against though 🤷‍♂️